src/components/Cryptocurrencies.jsx

```diff
@@ -3,54 +3,46 @@ import millify from 'millify';
 import { Link } from 'react-router-dom';
 import { Card, Row, Col, Input } from 'antd';
 
-import { useGetCryptosQuery} from '../services/cryptoApi';
+import { useGetCryptosQuery } from '../services/cryptoApi';
 
 
 const Cryptocurrencies = ({ simplified } ) => {
     const count = simplified ? 10 : 100;
     const { data: cryptosList, isFetching } = useGetCryptosQuery(count);
-    const [cryptos, setCryptos] = useState([]);
+    const [cryptos, setCryptos] = useState();
     const [searchTerm, setSearchTerm] = useState('');
 
     useEffect(() => {
+        setCryptos(cryptosList?.data.coins);
         const filteredData = cryptosList?.data?.coins.filter((coin) => coin.name.toLowerCase().includes(searchTerm.toLowerCase()));
 
         setCryptos(filteredData);
```

Comment:
You setCryptos twice here?  on line 16 and then again on line 19?  I'm not sure if line 16 is needed?

Also, you could do something you like

```
if (cryptosList?.data?.coins) {
   const filteredData = cryptosList?.data?.coins.filter((coin) => 
      coin.name.toLowerCase().includes(searchTerm.toLowerCase())
   );
   setCryptos(filteredData);
}